### DISCUSSION TITLE
Lighthouse: How to understand why the results are different on different platforms

### DISCUSSION THREAD

**Discussion by nonoumasy** [vercel/next.js#24972] [Help]
package.json
```
"dependencies": {
		"@firebase/rules-unit-testing": "^1.2.11",
		"@hookform/resolvers": "^1.0.1",
		"@material-ui/core": "^4.11.3",
		"@material-ui/icons": "^4.11.2",
		"@material-ui/styles": "^4.11.3",
		"clsx": "^1.1.1",
		"firebase": "^8.3.2",
		"jest": "^26.6.3",
		"material-ui-search-bar": "^1.0.0",
		"next": "10.1.2",
		"next-seo": "^4.24.0",
		"next-sitemap": "^1.6.29",
		"next-translate": "^1.0.6",
		"react": "17.0.2",
		"react-cool-onclickoutside": "^1.6.1",
		"react-dom": "17.0.2",
		"react-firebase-hooks": "^3.0.4",
		"react-hook-form": "^6.11.0",
		"react-masonry-css": "^1.0.16",
		"react-share": "^4.4.0",
		"react-slick": "^0.28.1",
		"react-social-login-buttons": "^3.4.0",
		"react-tiny-fab": "^4.0.2",
		"slick-carousel": "^1.8.1",
		"swr": "^0.5.6",
		"use-places-autocomplete": "^1.9.1",
		"yup": "^0.29.3"
	},
	
next.config.js

module.exports = {
    i18n: {
        locales: ['ar', 'de', 'en', 'es', 'fr', 'hi-IN', 'ja', 'pt', 'ru', 'zh' ],
        defaultLocale: 'en',
    },
    images: {
        domains: ['img.youtube.com', 'firebasestorage.googleapis.com', 'cloudinary.com', 'res.cloudinary.com'],
    },
}  	
```

Also, I am using Internationalized Routing path routing .
Just an observation, running the lighthouse again and again get different results. Sometimes the mobile will be 74 and then 81 another time and yet again at 87.
I'm using incognito when auditing these pages.
I understand that you want to run lighthouse incognito as well as test against the production build and NOT the dev server because that injects extra code which affects performance. So that explains the low score of 35 vs 95.

I'm getting 95 on desktop performance on Lighthouse on production (hacktravels.com):
![Screen Shot 2021-05-10 at 5 13 43 PM](https://user-images.githubusercontent.com/6312676/117741003-54373300-b1b6-11eb-8310-74fb9c0f316a.png)

I'm getting 35 on desktop performance Lighthouse on development:
![Screen Shot 2021-05-10 at 5 13 36 PM](https://user-images.githubusercontent.com/6312676/117741043-58fbe700-b1b6-11eb-9502-e5d77e9baaea.png)

I'm getting 74 on mobile performance Lighthouse for production:
![Screen Shot 2021-05-10 at 5 31 34 PM](https://user-images.githubusercontent.com/6312676/117741053-5c8f6e00-b1b6-11eb-88c9-29ad9e77da0c.png)

Also, on the Vercel dashboard: I'm getting 48. I'm not sure how this actually equates to Lighthouse metrics but the result shows poor performance on desktop vs the glowing 95 which lighthouse gives.
![Screen Shot 2021-05-10 at 5 16 40 PM](https://user-images.githubusercontent.com/6312676/117741060-5f8a5e80-b1b6-11eb-842a-48c57fd39451.png)

on Page Insights dev desktop 96:
![Screen Shot 2021-05-10 at 6 06 29 PM (2)](https://user-images.githubusercontent.com/6312676/117742880-8cd90b80-b1ba-11eb-8bd4-646134bbeddb.png)

on Page Insights dev mobile 79:
![Screen Shot 2021-05-10 at 6 06 33 PM (2)](https://user-images.githubusercontent.com/6312676/117742904-9a8e9100-b1ba-11eb-9407-a52ccdabacc6.png)

Not sure why the metrics are different on these different platforms. Is there a way to adjust this somehow?

**Comment by Sangdt** [comment]
First of all the score in your dev environment(localhost) isnt accurate since u only call to your local server and that is very different from the production server your app deploy on.
Second thing is lighthouse score in your chrome browser is only based on your machine and your internet speed not the same with everyone even if u having the same config with the google lighthouse page, it's still two difference things and Vercel analytic have different metric,calculations with google lighthouse.

So in order to get the correct score you should only use the production domain.
